### PR TITLE
docs: First InferenceService instructions with port-forwarding

### PR DESCRIPTION
**Pull request by nimishamehta5**
Adds a small section to the Getting Started > First InferenceService docs on how to curl against a locally running port-forwarded server.

### CODE REVIEW

File: docs/get_started/first_isvc.md [hunk 1]
```diff
@@ -186,6 +186,16 @@ Depending on your setup, use one of the following commands to curl the `Inferenc
...
     ```
 
+=== "Port forwarded service"
+
+    If you have done `Port Forward` above during testing, you can extract the inference
+    server URL from the status and make a curl request to it.
+
+    ```bash
+    SERVICE_HOSTNAME=$(kubectl get inferenceservice sklearn-iris -n kserve-test -o jsonpath='{.status.url}' | cut -d "/" -f 3)
+    curl -v -H "Host: ${SERVICE_HOSTNAME}" -H "Content-Type: application/json" "http://${INGRESS_HOST}:${INGRESS_PORT}/v1/models/sklearn-iris:predict" -d @./iris-input.json
```
Comment (spolti) [inline docs/get_started/first_isvc.md:196]: Maybe instead adding a new section, what about add the command curl command after https://github.com/kserve/website/pull/412/files#diff-2066f8cd5b8985c4a57b1da95f0b1da8277507031b72a35759bd066b47d11ee3R121

btw, is think there is no need to pass the SERVICE_HOSTNAME

Comment (nimishamehta5) [reply docs/get_started/first_isvc.md]: I updated the PR to add the curl command where you suggested. I think I had to modify the SERVICE_HOSTNAME section to work correctly.

**Comment by sivanantha321** [comment]
@nimishamehta5 The preview is not rendering the page. Can you check if it is rendering for you in your local machine ?
Screenshot from 2024-11-07 10-10-45

**Comment by nimishamehta5** [comment]
@sivanantha321 I think that's a different issue unrelated to my change, I reported this in https://github.com/kserve/website/issues/411 and it was fixed, but my branch is not rebased on to the main one

**Pull request merged**